### PR TITLE
Added script timeout option

### PR DESCRIPTION
**Pull request by kunagpal**
Sort of fixes https://github.com/postmanlabs/newman/issues/1189

### CODE REVIEW

File: lib/runner/extensions/event.command.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ module.exports = {
                 this.host.execute(event, {
                     id: executionId,
                     // debug: true,
-                    timeout: payload.scriptTimeout,
+                    timeout: payload.scriptTimeout || _.get(this.options, 'timeout.script'),
```
Comment (shamasis) [inline lib/runner/extensions/event.command.js]: Need to be _.has payload.scriptTimeout ? Payload : options

Comment (shamasis) [reply lib/runner/extensions/event.command.js]: add docs to payload

Comment (shamasis) [reply lib/runner/extensions/event.command.js]: add a todo to allow individual script level timeout

Comment (shamasis) [reply lib/runner/extensions/event.command.js]: need to follow timeout inheritence like that of request

```
timeout: _.min([
            _.get(options, 'timeout.request'),
            _.get(options, 'timeout.global')
        ]), // validated later inside requester
```

and maybe part config done in init of createContext of sandbox

**Comment by kunagpal** [comment]
Blocked on a patch release of https://github.com/postmanlabs/postman-sandbox/pull/193

File: lib/runner/extensions/event.command.js [hunk 2]
```diff
@@ -111,6 +111,7 @@ module.exports = {
         }
 
         sandbox.createContext(_.merge({
+            timeout: _(run.options.timeout).pick(['script', 'global']).values().min()
```
Comment (shamasis) [inline lib/runner/extensions/event.command.js]: fancy! :-)

Comment (kunagpal) [reply lib/runner/extensions/event.command.js]: I was probably in the Python one liner mode here. 😄

**Pull request merged**